dotnet-desktop-guide/net/winforms/compiler-messages/wfo0003.md

```diff
@@ -1,16 +1,20 @@
 ---
-title: Compiler Warning WFO5003
-description: Learn more about compiler warning WFO5003. ColorMode is for evaluation purposes only and subject to change.
+title: Compiler Warning WFO0003
+description: Learn more about compiler warning WFO0003. Unsupported high DPI configuration.
 ms.date: 11/19/2024
 f1_keywords:
-  - "WFO5001"
+  - "WFO0003"
 helpviewer_keywords:
-  - "WFO5001"
+  - "WFO0003"
 ---
 
-# Compiler Warning WFO5003
+# Compiler Warning WFO0003
 
-Remove high DPI settings from app.manifest and configure via Application.SetHighDpiMode API or 'ApplicationHighDpiMode' project property.
+**Version introduced:** .NET 9
+
+> Remove high DPI settings from app.manifest and configure via `Application.SetHighDpiMode` API or `ApplicationHighDpiMode` project property.
```

Comment:
Do you need a block quote `>` here?

Reply:
Could you please make  `Application.SetHighDpiMode`  and `ApplicationHighDpiMode` xref links to the APIs?